app/policies/school_cohort_policy.rb

```diff
@@ -1,6 +1,10 @@
 # frozen_string_literal: true
 
 class SchoolCohortPolicy < ApplicationPolicy
+  def index?
+    user.admin?
+  end
```

Comment:
Can we have a test for it?

Reply:
Good spot, actually this is not needed now so I will remove it.